### PR TITLE
Sc 300/add csv upload input to seed details stage

### PR DESCRIPTION
**Pull request by testing-life**
image

needs to be revisited after BE adds aan allow list param
```
const csvContent = this.csv && await this.csv[0].text();
    // for BE adds allow list param
    // this.launchConfig.launchDetails.allowList = csvContent;
```

with fake label
image

### CODE REVIEW

File: src/newLaunch/baseStage.scss [hunk 1]
```diff
@@ -37,9 +37,11 @@
   .content-block {
```
Comment (hiaux0) [inline src/newLaunch/baseStage.scss:37]: Can you please revert all unrelated formatting changes?

Comment (testing-life) [reply src/newLaunch/baseStage.scss:37]: done

File: src/newLaunch/seed/stages/stage4.html [hunk 2]
```diff
@@ -120,6 +120,22 @@
...
+                type="file"
+                accept=".csv"
+                files.bind="csv"
+                placeholder="Upload your .csv allowlist..." />
+              </div>
+            </span>
+              <div >
+                <button class="upload-csv" click.delegate="parseCsv()">Upload</button>
+                </div>
+          </div>
```
Comment (hiaux0) [inline src/newLaunch/seed/stages/stage4.html:138]: formatting off for this whole section, can you adjust please?

File: src/newLaunch/seed/stages/stage4.html [hunk 3]
```diff
@@ -120,6 +120,22 @@
...
+                <input class="footer-input"
+                id="allowList"
+                type="file"
+                accept=".csv"
+                files.bind="csv"
+                placeholder="Upload your .csv allowlist..." />
+              </div>
+            </span>
+              <div >
+                <button class="upload-csv" click.delegate="parseCsv()">Upload</button>
```
Comment (hiaux0) [inline src/newLaunch/seed/stages/stage4.html:136]: I expected to have have the file upload popup appear when pressing on "Upload".
(currently popup appears when clicking on the input)

Can we have an UX like this
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file

I added 2 Tasks to the ticket
https://app.shortcut.com/curvelabs/story/300/add-csv-upload-input-to-seed-details-stage-4

**Review by hiaux0 — COMMENTED** [review]
## Fnality
- Should be on 2 lines (label and input)
image

File: src/newLaunch/baseStage.scss [hunk 4]
```diff
@@ -101,9 +134,18 @@
   background-color: rgba(64, 52, 83, 1);
   color: $White;
   cursor: pointer;
+}
+
+.make-admin-button {
```
Comment (giladt) [inline src/newLaunch/baseStage.scss:139]: I suggest we follow only one naming convention. I guess camelCase is should be used across `prime-launch`.

**Review by giladt — CHANGES_REQUESTED** [review]

File: src/newLaunch/baseStage.scss (hunk 4 above)
Comment (testing-life) [reply src/newLaunch/baseStage.scss:139]: there seems to be a mix of naming conventions used - kebab case, some bem naming, camel case. this change was introduced in july
image

this specific class `.make-admin-button` was also introduced by someone else, also in july. my point is, your request would entail a refactor, which we do not want to do right now, correct ?

File: src/newLaunch/seed/stages/stage4.html (hunk 2 above)
Comment (testing-life) [reply src/newLaunch/seed/stages/stage4.html:138]: how about we move and do as giald suggests: 
_At this point probably best to not having any formatter, IMO… or once dev is merged into 756, we can create a PR that format’s all files to specific config and continue with unified EditorConfig from there?_
I don't know what settings to use to format it to your expectations

File: src/newLaunch/seed/stages/stage4.html (hunk 3 above)
Comment (testing-life) [reply src/newLaunch/seed/stages/stage4.html:136]: same mdn article displays this pattern, which is closer to our behaviour (minus the browser button):
image

File: src/newLaunch/baseStage.scss (hunk 4 above)
Comment (giladt) [reply src/newLaunch/baseStage.scss:139]: Make sense. Probably should be done together (or alongside) the formatting task you've suggested in the other comment...

Comment (testing-life) [reply src/newLaunch/baseStage.scss:139]: :+1:

**Review by hiaux0 — APPROVED** [review]
